src/pages/Wallet/AddressDetailsPage.tsx

```diff
@@ -126,6 +174,31 @@ const AddressDetailsPage = () => {
   )
 }
 
+const IOList = ({ currentAddress, isOut, outputs, inputs, timestamp }: IOListProps) => {
```

Comment:
I am reusing the component of the wallet home page, which will eventually be adjusted, and this will move to a new component to keep the code dry.